src/styles/page.css

```diff
@@ -3,6 +3,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  padding-bottom: 20px;
```

Comment:
Can you find a percentage value for this? A static value like this will have different renderings depending on screen size.

`padding-bottom: x%`